load_testing/common_flows/flow_sign_in.py

```diff
@@ -50,3 +49,94 @@ def do_sign_in(context):
     )
 
     return resp
+
+
+def do_sign_in_user_not_found(context):
+    num_users = get_env("NUM_USERS")
+    credentials = random_cred(num_users)
+
+    resp = do_request(context, "get", "/", "/")
+    auth_token = authenticity_token(resp)
+
+    if "/account" in resp.url:
+        print("You're already logged in. Quitting sign-in.")
+        return resp
+
+    # Post login credentials
+    resp = do_request(
+        context,
+        "post",
+        "/",
+        "/",
+        {
+            "user[email]": "sign_in_bad_user@example.com",
```

Comment:
should we add some randomness in here just in case somebody makes one of these accounts on accident in the load testing environemnt? ex `sign_in_bad_user+{randint}@example.com`

Reply:
Good idea - Changed this to use the random user's email prefixed with `actually-not-`